docker-compose.yml

```diff
@@ -15,9 +15,13 @@ services:
       - POSTGRES_DB=pgintegration,pgisintegration
       - ODC_DATACUBE_INDEX_DRIVER=postgres
       - ODC_POSTGIS_INDEX_DRIVER=postgis
+      - ODC_PSYCOPG31_INDEX_DRIVER=postgres
+      - ODC_PSYCOPG32_INDEX_DRIVER=postgis
       - ODC_NOSUCHDRIVERENV_INDEX_DRIVER=no_such_driver
       - ODC_DATACUBE_DB_URL=${ODC_DATACUBE_DB_URL:-postgresql://opendatacube:opendatacubepassword@postgres:35432/pgintegration}
       - ODC_POSTGIS_DB_URL=${ODC_POSTGIS_DB_URL:-postgresql://opendatacube:opendatacubepassword@postgres:35432/pgisintegration}
+      - ODC_PSYCOPG31_DB_URL=${ODC_PSYCOPG31_DB_URL:-postgresql+psycopg://opendatacube:opendatacubepassword@postgres:35432/pgintegration}
+      - ODC_PSYCOPG32_DB_URL=${ODC_PSYCOPG32_DB_URL:-postgresql+psycopg://opendatacube:opendatacubepassword@postgres:35432/pgisintegration}
       - ODC_NOSUCHDRIVERENV_DB_URL=${ODC_NOSUCHDRIVERENV_DB_URL:-postgresql://opendatacube:opendatacubepassword@postgres:35432/nosuchdriverenv}
```

Comment:
I'd prefer environment names a bit more transparent than "psycopg31" and "psycopg32".

Suggest "psycopg3gres" and "psycopg3gis"?

Reply:
Fixed!